spec/requests/api/v1/users_spec.rb

```diff
@@ -0,0 +1,176 @@
+require 'rails_helper'
+
+RSpec.describe API::V1::UsersController, type: :request do
+  describe "POST #create" do
+    let(:path) { "/v1/users" }
+
+    before { allow(SecureRandom).to receive(:uuid).and_return("password") }
+
+    context "with valid params" do
```

Comment:
It'd be great to make a shared rspec resource for some of the standard things we'll always be checking such as statuses, creates with valid params, etc.

Reply:
This isnt necessary here, but let's use the next endpoint as an excuse to do so.